### PR TITLE
Scene detail wall cell

### PR DESCRIPTION
**Pull request by Lv-215iOS**

### CODE REVIEW

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
+//
+//  FloorCell.swift
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift]: Wallcell

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+//  Sokoban
+//
+//  Created by admin on 1/18/17.
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift]: Who is admin?

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+//
+//
+
+import UIKit
+
+class FloorCell: UIView {
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift]: Why file name is Wallcell but class name is FloorCell?

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+    override func draw(_ rect: CGRect) {
+        let context = UIGraphicsGetCurrentContext()
+
+        context!.setFillColor(UIColor(red: 0.4392, green: 0.4118, blue: 0.4118, alpha: 1.0).cgColor)
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift]: !

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        context?.fill(rect)
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift:17]: ?

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+     
+        context?.setLineWidth((rect.width + rect.height)/100)
+        context?.setStrokeColor(UIColor(red: 0.2863, green: 0.0431, blue: 0, alpha: 1.0).cgColor)
+        context?.move(to: CGPoint(x: 0, y: 2 * rect.height/4))
+        context?.addLine(to: CGPoint(x: rect.width, y: 2 * rect.height / 4))
+        context?.strokePath()
+        
+        context?.setLineWidth((rect.width + rect.height)/100)
+        context?.setStrokeColor(UIColor(red: 0.2863, green: 0.0431, blue: 0, alpha: 1.0).cgColor)
+        context?.move(to: CGPoint(x: 0, y: 3 * rect.height/4))
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift:39]: Consider using some round function.

**Review by Lv-215iOS — COMMENTED** [review]
Please rename class inside. Everything else looks +/- good.

File: Sokoban/Classes/Controllers/Scene/WallCell.swift [hunk 7]
```diff
@@ -2,18 +2,18 @@
 //  FloorCell.swift
 //  Sokoban
 //
-//  Created by admin on 1/18/17.
+//  Created by Oleksandr on 1/18/17.
```
Comment (Lv-215iOS) [inline Sokoban/Classes/Controllers/Scene/WallCell.swift]: Who is Oleksandr?

**Review by sofiialushchanets — APPROVED** [review]

**Review by NazarKur — APPROVED** [review]
Almost good

**Review by pavlotsybulskyy — APPROVED** [review]
everything is OK

**Review by barabashd — APPROVED** [review]
Good job

**Review by oleksiib-huma — APPROVED** [review]

**Review by AlexDudash6 — APPROVED** [review]

**Review by oleksandryevdokymov — APPROVED** [review]
Fixed

**Pull request merged**